tests/test_watch_nmdc.py

```diff
@@ -13,54 +21,229 @@ def mock_cromwell(requests_mock, test_data_dir):
     requests_mock.post(cromwell_url, json=data)
     afile_path = test_data_dir / "afile"
     bfile_path = test_data_dir / "bfile"
-    metadata = {'outputs': {
-          "nmdc_rqcfilter.filtered_final": str(afile_path),
-          "nmdc_rqcfilter.filtered_stats_final": str(bfile_path),
-          "nmdc_rqcfilter.stats": {
```

Comment:
For all activities I believe it is a stats file that needs to be read in, not anything where cromwell metadata has numeric values. The variable names currently don't match up but i have a ticket for that to be fixed in with Kaitlyn and Valerie. 
Example of what this *should* look like is 
cfile_path=test_data_dir/"cfile"
where cfile is
{"input_read_count": 11431762,
            "input_read_bases": 1726196062,
            "output_read_bases": 1244017053,
            "output_read_count": 8312566
            }

Reply:
Removing this fixture.  Will implement it correctly in https://github.com/microbiomedata/nmdc_automation/issues/265